.github/actions/validate-issue/index.js

```diff
@@ -0,0 +1,121 @@
+    const pluginVersionsRegex = "pluginVersions|Plugin Version:";
+
+    const sfVersion = body.match(sfVersionRegex)?.[1];
+    const sfdxVersion = body.match(sfdxVersionRegex)?.[1];
+    const pluginVersionsIncluded = body.match(pluginVersionsRegex);
+
+    if ((sfVersion || sfdxVersion) && pluginVersionsIncluded) {
+      // FUTURE TODO:
+      // - Check for bundled plugins that are user installed (user) or linked (link)
+      // - Could do a check to see if the users has a prerelease version installed
+
+      if (sfVersion && sfVersion.startsWith("1.")) {
+        // TODO: Eventually suggest using sf@v2
+        const { old, latest } = compareVersions("@salesforce/cli", sfVersion);
+
+        if (old) {
+          const oldSf = getFile("./messages/old-cli.md", { THE_USER: issue.user.login, USER_CLI: "sf", USER_VERSION: sfVersion, LATEST_VERSION: latest });
+          postComment(oldSf);
+          addMoreInfoLabel();
+        }
+      }
+      if (sfdxVersion && sfdxVersion.startsWith("7.")) {
+        // TODO: Eventually suggest using sf@v2
+        const { old, latest } = compareVersions("sfdx-cli", sfdxVersion);
+
+        if (old) {
+          const oldSfdx = getFile("./messages/old-cli.md", { THE_USER: issue.user.login, USER_CLI: "sfdx", USER_VERSION: sfdxVersion, LATEST_VERSION: latest });
+          postComment(oldSfdx);
+          addMoreInfoLabel();
+        }
+      }
+
+      return;
+    } else {
+      const message = getFile("./messages/provide-version.md", { THE_USER: issue.user.login });
+      postComment(message);
+      addMoreInfoLabel();
+    }
+
+    // ---------
+    // FUNCTIONS
+    // ---------
+    async function postComment(body) {
+      const { data: comments } = await octokit.rest.issues.listComments({ owner, repo, issue_number });
+
+      // Check that this comment has not been previously commented
+      if (comments.length) {
+        if (comments.some((comment) => comment.body === body)) {
+          console.log("Already commented");
+          return;
+        }
+      }
+
+      return octokit.rest.issues.createComment({ owner, repo, issue_number, body });
+    }
+
+    function addMoreInfoLabel() {
+      return octokit.rest.issues.addLabels({
+        owner,
+        repo,
+        issue_number,
+        labels: ["more information needed"],
+      });
+      return;
+    }
+
+    function compareVersions(plugin, installed) {
+      const distTags = execSync(`npm view ${plugin} dist-tags --json`).toString();
+      const latest = JSON.parse(distTags).latest;
+      const old = semver.lt(installed, latest);
+
+      return {
+        old,
+        latest,
+      };
+    }
+
+    function getFile(filename, replacements) {
+      const contents = fs.readFileSync(path.join(__dirname, filename), "utf8");
+      return Object.entries(replacements || {}).reduce((acc, [key, value]) => {
+        return acc.replace(new RegExp(key, "g"), value);
```

Comment:
new js fun! 
```suggestion
        return acc.replaceAll(key, value);
```

FWIW, I wouldn't use reduce for this.  Just loop over the replacement key/value and call replaceAll, return the final,  since the `reduce` is mutating the contents anyway.